sdk/python/lib/pulumi/runtime/known_types.py

```diff
@@ -182,56 +195,56 @@ def new_file_asset(*args: Any) -> Any:
     """
     Instantiates a new FileAsset, passing the given arguments to the constructor.
     """
-    return _file_asset_resource_type(*args)
+    return _file_asset_resource_type(*args) # type: ignore
 
 
 def new_string_asset(*args: Any) -> Any:
     """
     Instantiates a new StringAsset, passing the given arguments to the constructor.
     """
-    return _string_asset_resource_type(*args)
+    return _string_asset_resource_type(*args) # type: ignore
```

Comment:
why?

Reply:
`_string_asset_resource_type` is an `Optional[type]` that gets filled in at runtime as far as I understand. The type checked complains that `None` isn't callable. We could probably make this go away by wrapping each of these in a `if _var not None:` check.